gnocchi/opts.py

```diff
@@ -176,16 +176,7 @@ def list_opts():
                 type=uuid.UUID,
                 help='Resource UUID to use to identify statsd in Gnocchi'),
             cfg.StrOpt(
-                'user_id',
-                deprecated_for_removal=True,
-                help='User ID to use to identify statsd in Gnocchi'),
-            cfg.StrOpt(
-                'project_id',
-                deprecated_for_removal=True,
-                help='Project ID to use to identify statsd in Gnocchi'),
-            cfg.StrOpt(
                 'creator',
-                default="${statsd.user_id}:${statsd.project_id}",
```

Comment:
the config file. it's `[statsd]user_id`. oslo.config supports variable substitution